### PR TITLE
More fixes for the details design

### PR DESCRIPTION
**Pull request by jancborchardt**

### CODE REVIEW

File: templates/detailItems/adr.html [hunk 1]
```diff
@@ -2,10 +2,6 @@
 	<option ng-repeat="option in ctrl.availableOptions" value="{{option.id}}">{{option.name}}</option>
 </select>
 <div>
-	<label for="details-pobox">{{ctrl.t.poBox}}</label>
```
Comment (DeepDiver1975) [inline templates/detailItems/adr.html:5]: @jancborchardt please leave the po box in here - it's a required part of the address

https://tools.ietf.org/html/rfc6350#section-6.3.1

Comment (DeepDiver1975) [inline templates/detailItems/adr.html:5]: there is even one more we need to add - the second ' the extended address (e.g., apartment or suite number);'

Comment (jancborchardt) [inline templates/detailItems/adr.html:5]: Ugh, way to make a spec complicated. :\ P.O. box won’t be used in the majority of cases anyway, similar to extended address.

**Comment by DeepDiver1975** [comment]
30db794 fixes #98

**Comment by jancborchardt** [comment]
kk, let’s merge?

**Comment by jancborchardt** [comment]
Ah ok, Travis is required here as well, sorry. :)

**Comment by DeepDiver1975** [comment]
> Ah ok, Travis is required here as well, sorry. :)

travis is setup to execute phpunit tests - we have no php code in here ...

**Comment by jancborchardt** [comment]
Still merging was seemingly blocked for me.

**Comment by DeepDiver1975** [comment]
> Still merging was seemingly blocked for me.

sure .... you are not an admin :wink:

**Comment by DeepDiver1975** [comment]
we need to sort out the ci situation in here ... next meetup maybe ...

**Pull request merged**